domain/src/test/java/org/oppia/domain/topic/TopicControllerTest.kt

```diff
@@ -623,6 +623,14 @@ class TopicControllerTest {
     assertThat(conceptCardLiveData.value!!.isFailure()).isTrue()
   }
 
+  @Test
+  fun testGetReviewCard_fractionSubtopicId1_isSuccessful() {
+    val reviewCardLiveData = topicController.getReviewCard(FRACTIONS_TOPIC_ID,"1")
+    val reviewCardResult = reviewCardLiveData.value
+    assertThat(reviewCardResult).isNotNull()
+    assertThat(reviewCardResult!!.isSuccess()).isTrue()
```

Comment:
Also, please check the content inside the review card too.

Reply:
+1

Reply:
added.